addons/mail/static/src/discuss/call/common/infinite_mirroring_warning.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="discuss.InfiniteMirroringWarning">
+        <div class="d-flex justify-content-start align-items-start position-fixed top-0 start-0 w-100 h-100 pe-none">
+            <div class="d-flex bg-dark rounded justify-content-between text-white fs-3 w-100 m-5 p-3 pe-auto">
+                <div class="d-flex align-items-center">
+                    Please share a specific window or tab or another monitor to avoid the infinite mirror effect.
```

Comment:
The color is good in white theme, but in dark theme the `bg-dark` becomes white and `text-white` is dark. This goes against the call view being dark.

The call view is a bit particular: we don't want the color to be theme specific. So make sure to have a same color scheme there in both white and dark theme.